### ISSUE TITLE
"Connection closed" on sleep

### ISSUE THREAD

**Issue by mtruyens** [thheller/shadow-cljs#582]
When I wake my Mac from sleep, I consistently get a "connection closed" error in ShadowCLJS. (I believe this was not the case some releases ago.)

Unfortunately, this means that I have to reload the entire browser session to get back to the point where I used to be before I put the Mac to sleep...

**Comment by thheller** [comment]
Hmm I did change the heartbeat behavior but it is more frequent now, so I'm not sure why it would affect sleep.

Did you upgrade your OS or Browser? Maybe the rules for sleep changed for some reason? I'll see if I can reproduce on my old macbook.

**Comment by mtruyens** [comment]
I follow the regular pace of browsers, so indeed frequently upgrade. I also recently upgraded my OS, but the "connection closed" issue already existed several weeks before. 

I just checked with other browsers, and apparently only the Chromium-derivatives (Chrome, new Microsoft Edge, Opera, Brave) have this problem — which are unfortunately my main development browsers, due to the CLJS devtools support. Firefox and Safari behave perfectly: the connection doesn't even close when the browser is another machine that is put to sleep.
